### DISCUSSION TITLE
Help with using an existing local duckdb database in shell.duckdb.org

### DISCUSSION THREAD

**Discussion by rmmales** [duckdb/duckdb#8037] [Q&A] [answered]
I have a local duckdb database (Windows), have been accessing using DBeaver.   I want to be able to get at it from https://shell.duckdb.org/.  I can use the .open command, it tells me:
Connected to a read-only remote database at: "C:\Projects\CSDP\SFBayStudy\Database\SFBayStudyCoastalDatabank.duckdb".

but at this point I don't seem to be able to do anything with it, for examples the tables command from CLI.  Have been unable to find a tutorial or reference.

Note that I can easily work with parquet files on AWS through the shell, just not the local database.

**Comment by carlopi** [comment, marked as answer]
Hi!
I think you may want to consider the `.files add someFile.duckdb`. It will open a file picker, you can select your own local file, and then using like `ATTACH someFile.duckdb; USE someFile.duckdb` you will now be using that file (or more precisely a copy of that file stored inside the browser environment).

Once you are done you can do `.files download someFile.duckdb` to download the file to your Download folder.

**Reply by rmmales** [reply]
Thank you, that worked.  I needed to handle some versioning problems with the database (needed to be upgraded to a more recent version), but after that I was able to follow your steps and get to what I wanted.
